### PR TITLE
[DALi][ACR-1597] Remove the whole dali guides

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Change Description ###

- DALi modules have become deprecated since Tizen 6.0 M2.
- Delete all guides related to DALi modules.

### Bugs Fixed ###

- N/A

### API Changes ###

 - ACR: ACR-1597

### CODE REVIEW

**Comment by Seoyeon2Kim** [comment]
Dear all reviewers,
Could you let me know what problems TizenDocsBot said? 
Then, I'll be able to fix them.

**Comment by annie-samsung** [comment]
> Dear all reviewers,
> Could you let me know what problems TizenDocsBot said?
> Then, I'll be able to fix them.

@Seoyeon2Kim, when you remove files, you have to check if the removed files where cross referenced from other documents and remove those links. Those are the documents having links to files you removed hence you have to remove.

**Comment by mijin85cho** [comment]
Hello, @Seoyeon2Kim 

1st comment is about the broken link.
The hyperlinks in index.md to the DALi pages were removed, and it triggered link checking of the links in the index.md
However, the link checking doesn't work well in index.md. Adding to that, the checked links are not related ones with DALi, but the whole links in index.md.
You can ignore it.

From the 2nd comment, these are suggestion to insert keywords to the pages, which will help to catch out easily what the main topic of each page is.
However as this PR is to remove the pages of deprecated DALi, you can ignore these comments also.

I hope @annie-samsung is going to review and approve & merge this PR.
As this PR is to remove the deprecated page, we don't need to check the contents inside,
so hope not taking much time.

P.S. Farewell, DALi.

**Comment by Seoyeon2Kim** [comment]
> Hello, @Seoyeon2Kim
> 
> 1st comment is about the broken link.
> The hyperlinks in index.md to the DALi pages were removed, and it triggered link checking of the links in the index.md
> However, the link checking doesn't work well in index.md. Adding to that, the checked links are not related ones with DALi, but the whole links in index.md.
> You can ignore it.
> 
> From the 2nd comment, these are suggestion to insert keywords to the pages, which will help to catch out easily what the main topic of each page is.
> However as this PR is to remove the pages of deprecated DALi, you can ignore these comments also.
> 
> I hope @annie-samsung is going to review and approve & merge this PR.
> As this PR is to remove the deprecated page, we don't need to check the contents inside,
> so hope not taking much time.
> 
> P.S. Farewell, DALi.

Apologies for the slow reply.
Oh, then I don't need to fix them yet here.
Thanks for the information!!! :)

File: docs/application/native/guides/applications/efl-app.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # EFL Applications
 
-You can use the Enlightenment Foundation Libraries (EFL) to create a 2D-based Tizen native application. However, EFL supports 2.5D and 3D effects and 3D objects as well. For 3D-based Tizen native applications, you can use the [Dynamic Animation Library (DALi) UI toolkit](../ui/dali/index.md) as well.
+You can use the Enlightenment Foundation Libraries (EFL) to create a 2D-based Tizen native application. However, EFL supports 2.5D and 3D effects and 3D objects as well.
```
Comment (nawab-samsung) [inline docs/application/native/guides/applications/efl-app.md:3]:
```suggestion
You can use the Enlightenment Foundation Libraries (EFL) to create a 2D-based Tizen native application. However, EFL supports 2.5D, 3D effects, and 3D objects as well.

File: docs/application/native/guides/applications/efl-app.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ With the EFL UI, you can create the following application types:
 
 - [Basic UI application](efl-ui-app.md)
 
-  A basic UI application provides a DALi-based user interface which allows the user to interact with the application.
+  A basic UI application provides a EFL-based user interface which allows the user to interact with the application.
```
Comment (nawab-samsung) [inline docs/application/native/guides/applications/efl-app.md:39]:
```suggestion
 A basic UI application provides an EFL-based user interface that allows the user to interact with the application.

File: docs/application/native/guides/applications/ui-app.md [hunk 3]
```diff
@@ -5,20 +5,14 @@ UI applications are Tizen native applications with a graphical user interface th
 
 A Tizen native UI application is similar to a conventional Linux application, with some additional features optimized for mobile and wearable devices. The additional features have constraints, such as a relatively small screen size and lack of system resources compared to a larger system. For example, for power management reasons, the application can take actions to reduce usage when it finds out that it has its display window covered over by another application window. State change events are delivered to make it possible to detect these situations.
 
-When creating a native UI application, you can select between EFL and DALi UI frameworks:
+When creating a native UI application, you can see EFL UI frameworks:
```
Comment (nawab-samsung) [inline docs/application/native/guides/applications/ui-app.md:8]: Is it see or select?
How about this?

 When creating a native UI application, you can use EFL UI frameworks:

File: docs/application/native/guides/ui/overview.md [hunk 4]
```diff
@@ -2,27 +2,14 @@
...
+The UI framework is a universal, reusable software environment that provides the essential building blocks as part of the Tizen platform to facilitate the development of Tizen native applications. The framework provides the window, UI components, and scene-based graphic rendering architecture needed to manage your application user interface. In addition, the framework provides animations and effects needed to make your application more fresh and vivid, event handling and the main loop to interact with the user and system, and other features, such as scalability and font setting. The Tizen platform provides the EFL UI toolkit frameworks and the Tizen window system shell.
```
Comment (nawab-samsung) [inline docs/application/native/guides/ui/overview.md:5]:
```suggestion
The UI framework is a universal, reusable software environment that provides the essential building blocks as part of the Tizen platform to facilitate the development of Tizen native applications. The framework provides the window, UI components, and scene-based graphic rendering architecture needed to manage your application user interface. 

In addition, the framework provides animations and effects needed to make your application more fresh and vivid. With event handling and using the main loop to interact with the user and system, it provides various other features, such as scalability and font setting. The Tizen platform provides the EFL UI toolkit frameworks and the Tizen window system shell for creating immersive user interfaces.

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
Please incorporate the comments. Thanks

**Comment by nawab-samsung** [comment]
@Seoyeon2Kim Request you to do needful.

**Comment by Seoyeon2Kim** [comment]
> @Seoyeon2Kim Request you to do needful.

Sorry for the late replay. I did other urgent task :( 
I'll update it today right away.

File: docs/application/native/guides/applications/efl-app.md (hunk 1 above)
Comment (Seoyeon2Kim) [reply docs/application/native/guides/applications/efl-app.md]: Done.

File: docs/application/native/guides/applications/efl-app.md (hunk 2 above)
Comment (Seoyeon2Kim) [reply docs/application/native/guides/applications/efl-app.md]: Done.

File: docs/application/native/guides/applications/ui-app.md (hunk 3 above)
Comment (Seoyeon2Kim) [reply docs/application/native/guides/applications/ui-app.md]: Actually, it is 'see', but I accepted your proposal. :)

File: docs/application/native/guides/ui/overview.md (hunk 4 above)
Comment (Seoyeon2Kim) [reply docs/application/native/guides/ui/overview.md]: Done.

**Review by nawab-samsung — APPROVED** [review]
As the contents of the files were pre-existing, I have checked the changed parts for grammar and sentence structure.

**Pull request merged**